l10n/en/firefox/features/private-browsing-2023.ftl

```diff
@@ -15,6 +15,7 @@ features-private-browsing-firefox-protects = { -brand-name-firefox } protects yo
 features-private-browsing-if-you-share-a-computer = If you share a computer with other people or if you want to limit how much data websites can collect about you, you can use private browsing mode in { -brand-name-firefox }. Private browsing erases the digital tracks you leave behind when you browse online — think of them like footprints through the woods.
 features-private-browsing-what-does-private-browsing-do = What does private browsing do?
 features-private-browsing-private-browsing-mode-opens-v2 = Private browsing mode opens a new browser window. When you close the last private browsing window, your browsing history and any tracking cookies from websites you visited will be erased. <strong>{ -brand-name-firefox } Pro Tip:</strong> Don’t forget to close all your private browsing windows when you’re done!
+# Obsolete string - can be removed once v2, above, is fully localized
```

Comment:
This should be accompanied by an expiry date as per: https://bedrock.readthedocs.io/en/latest/l10n.html#obsolete-strings

Reply:
:thumbsup: